### DISCUSSION TITLE
Text boxes don't offer selection on touch devices

### DISCUSSION THREAD

**Discussion by AArnott** [AvaloniaUI/Avalonia#15002] [Q&A]
A multi-line textbox works great on Windows. But the same UI on an Android device doesn't seem to support selection by tap and hold or dragging. And the context menu that comes up is in the wrong place (and never offers Copy, because selection appears to be impossible).
This doesn't feel like a native Android experience, and TBH is pretty crippling, making the app feel cheap. Is there anything that can be done to improve this?

**Comment by jp2masa** [comment]
What version are you using? There are text selection handles since 11.1.0-beta1: #13107.

**Reply by AArnott** [reply]
Ooh! That's exciting. I'm on 11.0.7 but I'll give the newer beta you suggest a try.

**Reply by AArnott** [reply]
Unfortunately, I'm blocked in testing it due to a crash the beta introduces: https://github.com/AvaloniaUI/Avalonia/issues/15009

**Reply by timunie** [reply]
You could at least test it in a minimal sample if it would solve your issue, once the other linked is solved as well,
